build_tools/circle/build_test_pypy.sh

```diff
@@ -21,10 +21,8 @@ which python
 pip install -U pip
 
 # pins versions to install wheel from https://antocuni.github.io/pypy-wheels/manylinux2010
-pip install --extra-index-url https://antocuni.github.io/pypy-wheels/manylinux2010 numpy==1.18.0 scipy==1.3.2
+pip install --extra-index-url https://antocuni.github.io/pypy-wheels/manylinux2010 Cython numpy==1.19.1 scipy==1.5.2
```

Comment:
The NumPy wheel is available [on PyPI](https://pypi.org/project/numpy/#files) (at the bottom of the page) now